badge/Badge/Badge.html

```diff
@@ -0,0 +1,339 @@
+<html>
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title> Alert </title>
+        <link rel="stylesheet" href="Badge.css" />
```

Comment:
Instead of importing individual files in your HTML file, import a single main file.

Creating specific css styles for each component and importing *only* that css in your HTML would not be a good way of architecting component library. Your component library needs to have single entry point - meaning that user should import only one link to use the components not link specific to component that he intends to use.

I'm using something like this.
``` 
index.html
(folder) styles / 
   style.css 
    /* this should be your main entry point. Add all your css for documentation site. 
        If there are classes that you want to specifically use for component library prefix them with some letters: 
        For eg:  consider you have a class named `container` that is being used only in documentation site, 
        prefix it with some thing like `cl` the class becomes `cl-container` */
   (folder) components / 
       - badge.css
       - alert.css
       - avatar.css
       .
       .
       . //so on
```

and my style.css file looks like

```css
import url("./components/badge.css");
import url("./components/alertcss");
import url("./components/avatar.css");



/* Other extra styles */
```